### PR TITLE
DTO and validator updated for DLQ

### PR DESCRIPTION
**Pull request by dawidkeshop**

### CODE REVIEW

File: src/CaptainHook.Application/Validators/Dtos/SubscriberDtoValidator.cs [hunk 1]
```diff
@@ -13,6 +13,9 @@ public SubscriberDtoValidator()
 
             RuleFor(x => x.Callbacks)
                 .SetValidator(new WebhooksDtoValidator("Callbacks"));
+
+            RuleFor(x => x.Dlq)
+                .SetValidator(new WebhooksDtoValidator("Callbacks"));
```
Comment (tabman83) [inline src/CaptainHook.Application/Validators/Dtos/SubscriberDtoValidator.cs:18]: I'd say DLQ here

**Review by tabman83 — CHANGES_REQUESTED** [review]

File: src/CaptainHook.Application/Validators/Dtos/SubscriberDtoValidator.cs (hunk 1 above)
Comment (dawidkeshop) [reply src/CaptainHook.Application/Validators/Dtos/SubscriberDtoValidator.cs]: sure

**Review by tabman83 — COMMENTED** [review]
May I suggest we call them DlqHooks to reinforce the concept they are still webhooks, just fired when an event ends up in DLQ? @chomick @dawidkeshop @mszymeshop @nikhilgirraj

File: src/CaptainHook.Application/Validators/Dtos/SubscriberDtoValidator.cs [hunk 2]
```diff
@@ -13,6 +13,9 @@ public SubscriberDtoValidator()
 
             RuleFor(x => x.Callbacks)
                 .SetValidator(new WebhooksDtoValidator("Callbacks"));
+
+            RuleFor(x => x.Dlq)
+                .SetValidator(new WebhooksDtoValidator("DlqHooks"));
         }
```
Comment (tabman83) [inline src/CaptainHook.Application/Validators/Dtos/SubscriberDtoValidator.cs:19]: Shouldnt this be DlqHooks as well?

**Review by tabman83 — APPROVED** [review]

File: src/CaptainHook.Application/Validators/Dtos/SubscriberDtoValidator.cs (hunk 2 above)
Comment (dawidkeshop) [reply src/CaptainHook.Application/Validators/Dtos/SubscriberDtoValidator.cs]: Of course it should, by mistake I just didn't save all changes in this file.

**Pull request merged**